.github/workflows/ci-terraform.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           fetch-depth: 2
       - id: workflow_files
         run: |
-          echo "::set-output name=files::$(git diff --name-only --diff-filter=ACMRT HEAD^ HEAD | grep -E '${{ env.workflow_files_re }}')"
+          echo "::set-output name=files::$(git diff --name-only --diff-filter=ACMRTD HEAD^ HEAD | grep -E '${{ env.workflow_files_re }}')"
```

Comment:
Similar to the feedback on `ci-java.yml`, I think the on condition needs to be updated, and the workflow job should be removed along with the dependencies.
```
on:
  pull_request:
    paths:
      - ops/terraform/**
```

Reply:
https://github.com/CMSgov/beneficiary-fhir-data/pull/773